src/state/slices/instructor/actions/index.js

```diff
@@ -1,37 +1,37 @@
-import * as InstructorScreenActionType from './InstructorScreenActionType';
+import * as instructorActionTypes from './instructorActionTypes'
 
 export function receiveInstructor(instructor) {
   return {
-    type: InstructorScreenActionType.RECEIVE_INSTRUCTOR,
+    type: instructorActionTypes.RECEIVE_INSTRUCTOR,
```

Comment:
I use capital case in the single tense to describe a container of static values. This might make me insane ;)